EstateAdministrationUI/Startup.cs

```diff
@@ -92,11 +110,37 @@ public void ConfigureServices(IServiceCollection services)
                                                                                                                                                   ValidateIssuer = false
                                                                                                                                               };
                                                                                                       });
+            IdentityModelEventSource.ShowPII = true;
+
+            services.AddSingleton<IApiClient, ApiClient>();
+            services.AddSingleton<IModelFactory, ModelFactory>();
+            services.AddSingleton<IViewModelFactory, ViewModelFactory>();
+            services.AddSingleton<IEstateClient, EstateClient>();
+            services.AddSingleton<Func<String, String>>(container => (serviceName) =>
+                                                                     {
+                                                                         var x = ConfigurationReader.GetBaseServerUri(serviceName).OriginalString;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'x'.](https://app.codacy.com/manual/stuart_ferguson1/EstateAdministrationUI/pullRequest?prid=5485852)